tools/sidekick/config.json

```diff
@@ -22,7 +22,7 @@
       "id": "localize",
       "title": "Localize",
       "environments": [ "edit" ],
-      "url": "https://main--milo--adobecom.hlx.page/tools/loc/index.html?project=bacom--adobecom",
+      "url": "https://main--bacom--adobecom.hlx.page/tools/loc/index.html?project=bacom--adobecom&&milolibs=loc",
```

Comment:
I'm curious about why milolibs is set to a feature branch rather than main

Reply:
This was because localization feature was moved under milo's `/libs` only in loc branch for now.